src/styles/blocks/product.scss

```diff
@@ -0,0 +1,70 @@
+.product {
+  display: flex;
+  flex-direction: column;
+  color: $main-font-color;
+
+  &__image {
+    background-image: url(/src/images/imac.jpeg);
+    height: 134px;
+    width: 160px;
+    background-size: 100% 100%;
+    margin-bottom: 40px;
+    margin-inline: auto;
```

Comment:
The image URL should not include the `/src/` part as it assumes a specific project structure that might not be true in all environments, especially after the project is built. Consider using a relative path from the CSS file to the image or defining paths as variables if they are used in multiple places.